src/features/auth/components/Register/RegistrationFooter.tsx

```diff
@@ -1,32 +1,37 @@
 import React from 'react';
 
 import { Container, Link, Text } from '@chakra-ui/react';
+import { Trans } from 'react-i18next';
 import { Link as RouterLink } from 'react-router-dom';
 
 import { PATHS } from '@/routes/paths';
 
 export const RegistrationFooter = () => (
   <Container w="110%" mt="1rem" px="0" mx="-0.5rem">
     <Text align="center">
-      By signing up, you accept our{` `}
-      <Link as={RouterLink} to={`/${PATHS.TERMS_AND_CONDITIONS}`} colorScheme="lightblue">
-        Conditions
-      </Link>
-      , the{' '}
-      <Link as={RouterLink} to={`/${PATHS.PRIVACY_POLICY}`}>
-        Data Policy
-      </Link>{' '}
-      and the{' '}
-      <Link as={RouterLink} to={`/${PATHS.COOKIES}`}>
-        Cookies Policy
-      </Link>
-      .
+      <Trans i18nKey="auth.register.termsAndConditionsText">
+        By signing up, you accept our
+        <Link as={RouterLink} to={`/${PATHS.TERMS_AND_CONDITIONS}`} colorScheme="lightblue">
+          Conditions
+        </Link>
+        , the
+        <Link as={RouterLink} to={`/${PATHS.PRIVACY_POLICY}`}>
+          Data Policy
+        </Link>
+        and the
+        <Link as={RouterLink} to={`/${PATHS.COOKIES}`}>
+          Cookies Policy
+        </Link>
+        .
```

Comment:
I think this should be:
```suggestion
        <Link as={RouterLink} to={`/${PATHS.TERMS_AND_CONDITIONS}`} colorScheme="lightblue" />
        <Link as={RouterLink} to={`/${PATHS.PRIVACY_POLICY}`} />
        <Link as={RouterLink} to={`/${PATHS.COOKIES}`} />
```

Reply:
also left a comment on how to change the translation file